shell/ci/testing/sync-package-hjson.js

```diff
@@ -66,7 +66,7 @@ if (match === null) {
 
         let githubToken;
         try {
-          githubToken = readFileSync(githubTokenPath);
+          githubToken = fs.readFileSync(githubTokenPath);
         } catch (err) {
           console.error("No GitHub token found:", err);
           process.exit();
```

Comment:
If no GitHub token is found, perhaps it *should* fail.
```suggestion
          process.exit(1);
```